README.md

```diff
@@ -109,6 +120,36 @@ import 'package:flutter_credit_card/flutter_credit_card.dart';
     ),
 ```    
 
+*Floating Card*
```

Comment:
I think a row of check boxes with indicating supported platform should also be added